target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -31,6 +31,9 @@ wnce2001|\
 zbt-we2026)
 	set_wifi_led "$boardname:green:wlan"
 	;;
+adslr,g7)
+	ucidef_set_led_netdev "wifi" "wifi" "$boardname:blue:sys" "eth0.2" "tx rx"
```

Comment:
You assigned wan trigger to sys led and named it "wifi". This is weird.
This kind of custom led usage (using sys led for other stuff) is usually done by users themselves. It shouldn't be applied via default settings. These three lines can be dropped.

Reply:
After deleting this option, the wifi light does not light up properly.

Reply:
If that's a wifi light you should name it "wifi" instead of "sys" and use wlan triggers for it.

Reply:
If that's just a status led without special labels you shouldn't add a netdev trigger for it.

Reply:
My original intention was because the WiFi light didn't light up after entering the system, so I thought of using the wan port as a trigger. When there is data to interact, it flashes.